backend/model_server/encoders.py

```diff
@@ -145,6 +121,41 @@ def _embed_vertex(
         )
         return embedding[0].values
 
+    @retry(tries=3, backoff=2, delay=5)
```

Comment:
This retry wraps all cloud embedding api calls. 

Known working fix for cohere. unsure about the rest